cpp/oneapi/dal/data/undirected_adjacency_array_graph.cpp

```diff
@@ -71,58 +58,38 @@ auto get_edge_count_impl(const G &g) noexcept -> edge_size_type<G> {
     return layout->_edge_count;
 }
 
-template edge_size_type<graph64> get_edge_count_impl(const graph64 &g);
 template edge_size_type<graph32> get_edge_count_impl(const graph32 &g);
 
 template <typename G>
 auto get_vertex_degree_impl(const G &g, const vertex_type<G> &vertex) -> vertex_edge_size_type<G> {
     const auto &layout = detail::get_impl(g);
-    return layout->_degrees[vertex];
+    return layout->_vertexes[vertex + 1] - layout->_vertexes[vertex];
 }
 
-template vertex_edge_size_type<graph64> get_vertex_degree_impl(const graph64 &g,
-                                                               const vertex_type<graph64> &vertex);
 template vertex_edge_size_type<graph32> get_vertex_degree_impl(const graph32 &g,
```

Comment:
another observation on the style, graph32 is a type name? If yes, why we do Graph in one place and graph32 in another place